ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__arm64-stable-4.16-upgrade-from-stable-4.15.yaml

```diff
@@ -1,6 +1,6 @@
 base_images:
   ansible:
-    name: "4.16"
+    name: "4.15"
```

Comment:
Is this change expected ?

Reply:
don't you say ansible is for installation purpose?

Reply:
Ansible stated here should be used for RHEL related installation.
We have ansible in upgrade, too. But that should be handled in the upgrade step.

Reply:
Hmm, weird, I remembered [the comment](https://github.com/openshift/release/pull/49710#discussion_r1531446627) was for upi-installer, but now seem like it was `ansible`

Reply:
Comment for upi-installer is [here](https://github.com/openshift/release/pull/49710#discussion_r1524105292), earlier in this PR.
There are several version changes in this PR.

Reply:
So what's the decision for ansible version? same with initial OCP version or target OCP version?

Reply:
Yes, rhel worker scaleup in install workflow need ansible, so I think the ansible version should match source cluster version. I also checked `ansible` rhel worker upgrade part in `cucushift-upgrade-toimage`, sounds like we did not care about ansible version too much, and no action to switch ansible verison, just using the installed one in verification-tests:latest image. So for now, the code looks good to me. cc @gpei to double confirm.

Reply:
In `cucushift-upgrade-toimage`, we do switch ansible based on OCP version,
https://github.com/openshift/release/blob/master/ci-operator/step-registry/cucushift/upgrade/toimage/cucushift-upgrade-toimage-commands.sh#L231-L236

Reply:
Ah, good point. So does that mean, multiple version of ansible are already installed in verification-tests:latest image, the rhel worker scale up function will control the switch of ansible by itself. So the above conclusion - "the ansible version should match source cluster version", "So for now, the code looks good" sounds valid, right?

Reply:
And `workers-rhel-scaleup` is using `from: ansible`
https://github.com/openshift/release/blob/master/ci-operator/step-registry/workers-rhel/scaleup/workers-rhel-scaleup-ref.yaml

Reply:
Yes, i agree "the ansible version should match source cluster version"

Reply:
Aha, just found this is changing from 4.16 to 4.15.  I've misread it as change 4.15 to 4.16

No issue for this line.

Reply:
Do you mean we never need set `ansible` base image here?

Reply:
yeah, for RHEL scaleup action, 4.15 ansible image is expected here. 
And for RHEL worker upgrade action, just as @liangxia pointed above, we controlled the ansible being used in `cucushift-upgrade-toimage` step based on the target OCP version.